### ISSUE TITLE
GUI and BSSH

### ISSUE THREAD

**Issue by sandrex** [iiordanov/BSSH-legacy#3]
I am connecting to a linux machine and when I try to run GUI Apps I have got some errors.
Example:

$ kcalc
Kcalc: cannot connect to X server

Is it possible to run graphical apps using BSSH?

**Comment by iiordanov** [comment]
No, it is not possible, because there is no X server on your Blackberry 10
device to draw the programs!

On Sun, Sep 15, 2013 at 6:46 PM, sandrex notifications@github.com wrote:

> I am connecting to a linux machine and when I try to run GUI Apps I got
> some errors.
> Example:
> 
> $ kcalc
> Kcalc: cannot connect to X server
> 
> Is it possible to run graphical apps using BSSH?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/iiordanov/BSSH/issues/3
> .

## 

The conscious mind has only one thread of execution.

**Comment by sandrex** [comment]
Thank you very much for the early reply.
